app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +21,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <service
+            android:name=".WeatherUpdateService"
+            android:enabled="true"
+            android:exported="false"/>
```

Comment:
По умолчанию он и так false, если нет intent-filter'ов

> The default value depends on whether the service contains intent filters. The absence of any filters means that it can be invoked only by specifying its exact class name. This implies that the service is intended only for application-internal use (since others would not know the class name). So in this case, the default value is "false". On the other hand, the presence of at least one filter implies that the service is intended for external use, so the default value is "true".